errors/errors.go

```diff
@@ -0,0 +1,48 @@
+package errors
+
+import (
+	"errors"
+	"log"
+	"os"
+)
+
+// Const may be used to convert constant strings into `error` values.
```

Comment:
This feels out of place in this package. To be honest, I was hoping this package would remain named `linux`.

Reply:
I.e. the package would have linux-specific errnos, and functions related to modifying those (e.g. ExtractErrno, as below), but generic error functions would be unrelated.

IMO I'd rather remove this from the change and keep using errors.New elsewhere.

Reply:
Sounds good to me.
I'm gonna try to atomize this into just exporting the package and changing the call sites.

Reply:
I do like having the new errors.Is/As -- I like that rewrite.

I can also get with the whole Decref -> Close -> return errors thing, though probably a separate PR?

Reply:
For sure.
I think as-is this chunk is a lot more sensible to look at.
I can submit the other parts as a follow-up.